demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -692,7 +692,8 @@ def validate_pack_unique_files(self, pack_path: str, pack_error_ignore_list: dic
                                                                skip_id_set_creation=self.skip_id_set_creation,
                                                                prev_ver=self.prev_ver,
                                                                json_file_path=self.json_file_path)
-        pack_errors = pack_unique_files_validator.are_valid_files()
+        pack_errors = pack_unique_files_validator.are_valid_files(self.id_set_validations,
+                                                                  pack_error_ignore_list)
```

Comment:
no need to pass the pack ignore list - it is passed in the `PackUniqueFilesValidator` setup.
internally simply use the `self.ignored_errors`.

Reply:
done